### PR TITLE
Specify particular versions for skipped/slow chrome-beta e2e tests

### LINKED ISSUE #5494: [e2e] Update e2e references which skip on chrome-beta to skip on chrome versions instead
#### Summary
At the moment, we're skipping on chrome-beta. We should update our test suite (in 2.1.0) to skip by browser version instead.

### PR DESCRIPTION
**Pull request by ozyx**
Closes #5494 

### Describe your changes:

There are a couple `chrome-beta` tests which we have marked as `fixme` (skip) or `slow` (doubles the timeouts). Previously we were marking these for every run with the `chrome-beta` project regardless of browser version. This change will specify the particular version of `chrome-beta` we wish to mark as skip/slow.

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [x] Is this change backwards compatible? For example, developers won't need to change how they are calling the API or how they've extended core plugins such as Tables or Plots.

### Author Checklist

* [x] Changes address original issue?
* [ ] Unit tests included and/or updated with changes?
* [x] Command line build passes?
* [x] Has this been smoke tested?
* [ ] Testing instructions included in associated issue OR is this a dependency/testcase change?

### Reviewer Checklist

* [ ] Changes appear to address issue?
* [ ] Changes appear not to be breaking changes?
* [ ] Appropriate unit tests included?
* [ ] Code style and in-line documentation are appropriate?
* [ ] Commit messages meet standards?
* [ ] Has associated issue been labelled unverified? (only applicable if this PR closes the issue)
* [ ] Has associated issue been labelled bug? (only applicable if this PR is for a bug fix)

### CODE REVIEW

File: e2e/tests/plugins/imagery/exampleImagery.e2e.spec.js [hunk 1]
```diff
@@ -181,8 +181,11 @@ test.describe('Example Imagery Object', () => {
 
     });
 
-    test('Can use the reset button to reset the image', async ({ page }, testInfo) => {
-        test.slow(testInfo.project === 'chrome-beta', "This test is slow in chrome-beta");
+    test('Can use the reset button to reset the image', async ({ page, browser }, testInfo) => {
+        test.slow(
+            testInfo.project === 'chrome-beta' && browser.version() === '104.0.5112.39',
```
Comment (unlikelyzero) [inline e2e/tests/plugins/imagery/exampleImagery.e2e.spec.js:186]: In this case, I think we'd want to check the version of chrome. It's possible that this performance regression will not be fixed and become mainline chrome.

I would set this test to slow if brower.version() > 103 && browser='chrome'

File: e2e/tests/plugins/imagery/exampleImagery.e2e.spec.js [hunk 2]
```diff
@@ -181,8 +181,11 @@ test.describe('Example Imagery Object', () => {
 
     });
 
-    test('Can use the reset button to reset the image', async ({ page }, testInfo) => {
-        test.slow(testInfo.project === 'chrome-beta', "This test is slow in chrome-beta");
+    test('Can use the reset button to reset the image', async ({ page, browser }, testInfo) => {
+        test.slow(
```
Comment (unlikelyzero) [inline e2e/tests/plugins/imagery/exampleImagery.e2e.spec.js:185]: Would you mind running this locally and specifying the chrome-canary config? You'll have to do a `npx playwright install chrome-canary` first

Comment (ozyx) [reply e2e/tests/plugins/imagery/exampleImagery.e2e.spec.js:185]: I'm unable to get this to work. `npx playwright install chrome-canary` is not valid, and Playwright fails to find canary google chrome on my Mac (even after force reinstalling), and it's unavailable in linux. Were you able to get this working locally at some point in the past?

**Comment by unlikelyzero** [comment]
Closing in favor of https://github.com/nasa/openmct/pull/5756

**Pull request closed without merging**